### PR TITLE
[MOD-2169] Fixed some bugs and cosmetic issues with ti.cloud sample app

### PR DESCRIPTION
**Pull request by wluu**

### CODE REVIEW

File: commonjs/example/app.js [hunk 1]
```diff
@@ -16,30 +16,30 @@ Cloud.debug = true;
```
Comment (ashcoding) [inline commonjs/example/app.js]: Something you didn't write but it seems like on line 11 and line 14, same code was written. Would be great if you could help remove either lines.

File: commonjs/example/samples/status/create.js [hunk 2]
```diff
@@ -40,6 +43,7 @@ exports['Create Status'] = function (evt) {
     if (Ti.Media.showCamera) {
         var takePhoto = Ti.UI.createButton({
             title: 'Take Photo with Camera',
+			color: '#000',
             top: 10 + Utils.u, left: 10 + Utils.u, right: 10 + Utils.u, bottom: 10 + Utils.u,
             height: 40 + Utils.u
         });
```
Comment (ashcoding) [inline commonjs/example/samples/status/create.js]: This code needs updating. With Android 6.0, camera requires permissions to be checked before show camera can be used.
Eg:
```javascript
if (Ti.Media.hasCameraPermissions()) {
    // Show camera code should be here
} else { 
    Ti.Media.requestCameraPermissions(function(e) {
             if (e.success === true) {
                 // Show camera code should be here
             } else {
                 alert("Access denied, error: " + e.error);
             }
    });
}
```

**Comment by ashcoding** [comment]
I feel that `hintTextColor` for Android shouldn't be `#000`. A user might mistake that the text is already written when in fact it's the hint. Instead, perhaps a grey color could be used.

File: commonjs/example/samples/users/search.js [hunk 3]
```diff
@@ -8,6 +8,8 @@ exports['Search User'] = function (evt) {
```
Comment (ashcoding) [inline commonjs/example/samples/users/search.js]: General comment on this. Using this api returns a 410 (Gone).
``` 
search.json has been deprecated. Please use $text query
```
Needs updating.

File: commonjs/example/samples/checkins/create.js [hunk 4]
```diff
@@ -60,7 +64,10 @@ exports['Create Checkin'] = function (evt) {
     function findMe() {
 
         table.setData([
-            { title: 'Geolocating...' }
+            { 
+            	title: 'Geolocating...',
+            	color: '#000' 
+            }
         ]);
```
Comment (ashcoding) [inline commonjs/example/samples/checkins/create.js]: Geolocation in Android 6.0 requires the Location permission. Similar to the Camera permission mentioned earlier.

Using "places/search.json" also returns 
```
410: search.json has been deprecated. Please use $text query
```

**Comment by ashcoding** [comment]
For the purpose of the Ticket MOD-2169, I think this PR is acceptable but a new ticket needs to be created to revisit and update the ti.cloud sample as certain endpoints have been deprecated as mentioned above.

@wluu What do you think?

**Comment by wluu** [comment]
@ashcoding Agreed! Later today, I'll update the PR with grey hintText and file another ticket to update ti.cloud sample app.

**Comment by wluu** [comment]
And, here is the ticket to update ti.cloud sample app: https://jira.appcelerator.org/browse/MOD-2170

**Comment by hansemannn** [comment]
Closing in favor of https://github.com/appcelerator-modules/ti.cloud/pull/94

**Pull request closed without merging**